include/lsst/afw/image/PhotoCalib.h

```diff
@@ -417,6 +417,13 @@ class PhotoCalib final : public table::io::PersistableFacade<PhotoCalib>, public
      */
     double getCalibrationErr() const { return _calibrationErr; }
 
+    /**
+     * Is this photoCalib spatially constant?
+     *
+     * @returns    Is the calibration spatially constant?
+     */
+    bool isConstant() const { return _isConstant; }
```

Comment:
Right now, unfortunately, the different calibrations don't behave exactly the same, at least if you're trying to vectorize the operations.  If you prefer I can expose this via pybind11 as `_isConstant` rather than `isConstant`.  I'll do that.

Reply:
Yes, `_isConstant` at least implies it's an internal thing. Thanks.